features/features.go

```diff
@@ -0,0 +1,96 @@
+//go:generate stringer -type=FeatureFlag
+
+package features
+
+import (
+	"expvar"
+	"fmt"
+	"sync"
+)
+
+type FeatureFlag int
+
+const (
+	unused                    FeatureFlag = iota
+	DNSAllowLoopbackAddresses             // Common
+	CheckMalformedCSR                     // WFE
+	DoNotForceCN                          // RA, CA
+	ReuseValidAuthz                       // RA
+	EnableMustStaple                      // CA
+	AcceptRevocationReason                // WFE
+)
+
+// List of features and their default value
```

Comment:
I think it would actually require extra code to only allow enabling features than allowing enable/disable as the bool feature map is still required.